src/event-interface.d.ts

```diff
@@ -0,0 +1,6 @@
+export default class EventInterface {
+	
+	extend():void;
+	getOnMethod():Function;
+	getFireMethod():Function;
```

Comment:
Missing `on` and `fire` methods:

```suggestion
	getFireMethod():Function;
	on(event:string, callback:Function):void;
	fire(event:string, data:object):void;
```